doc/source/advanced.rst

```diff
@@ -654,7 +654,7 @@ setting the index of a ``DataFrame/Series`` with a ``category`` dtype would conv
 
    df = pd.DataFrame({'A': np.arange(6),
                       'B': list('aabbca')})
-   df['B'] = df['B'].astype('category', categories=list('cab'))
+   df['B'] = df['B'].astype(pd.CategoricalDtype(list('cab')))
```

Comment:
I think i left a comment about this. but this ``CategoricalDtype`` is already in ``pandas.api.types``. do not add to the top level.

Reply:
Fixed in 1364f2d